File: tests/fixtures/vulnerable/eval_test.py
# aiscan test fixture — should trigger AI-SEC-009 (Eval/Exec)
# DO NOT USE IN PRODUCTION

user_input = ""  # imagine this comes from a web request

# Unsafe: eval on user input
result = eval(user_input)

# Unsafe: exec on user input
exec(user_input)

File: tests/fixtures/vulnerable/eval_test.js
// aiscan test fixture — should trigger AI-SEC-009 JS (Eval/Exec)
// DO NOT USE IN PRODUCTION

const userInput = req.body.code;

// Unsafe: eval on user input
const result = eval(userInput);

// Unsafe: new Function from user input
const fn = new Function('return ' + userInput);
